setup.sh

```diff
@@ -1,14 +1,14 @@
 #!/usr/bin/env bash
 
 cd
-if ! command -v papertest; then
+if ! command -v papertest > /dev/null 2>&1; then
     touch .paperdebug
     echo "setting up pb development environment"
     curl https://raw.githubusercontent.com/paperbenni/bash/master/debugger.sh >papertest
     chmod +x papertest
-    sudo mv papertest /usr/bin/
+    sudo mv papertest /usr/bin/ # <-- TODO: Check for safe ownership & permissions!
```

Comment:
I'd check this ASAP, as it's a potential security risk.

Reply:
Could you elaborate on that? I don't know a lot about permission security...